themes/digital.gov/layouts/partials/core/primary_communities.html

```diff
@@ -0,0 +1,43 @@
+{{ $mainCommunities := (where .Site.Pages "Section" "communities") }}
+{{ $mainCommunities = (where $mainCommunities ".Params.dg_highlight" "eq" true) }}
+{{ $featuredCommunity := (where $mainCommunities ".Params.weight" "eq" 2) }}
+{{ $mainCommunities = union $featuredCommunity $mainCommunities }}
```

Comment:
Add comments at top of page explaining what this partial does. Comments will help explain the what and why for the code.